sweep/tx_input_set.go

```diff
@@ -0,0 +1,80 @@
+package sweep
+
+import (
+	"github.com/btcsuite/btcutil"
+	"github.com/lightningnetwork/lnd/input"
+	"github.com/lightningnetwork/lnd/lnwallet/chainfee"
+)
+
+// txInputSet is an object that accumulates tx inputs and keeps running counters
+// on various properties of the tx.
+type txInputSet struct {
+	// weightEstimate is the (worst case) tx weight with the current set of
+	// inputs.
+	weightEstimate input.TxWeightEstimator
+
+	// inputTotal is the total value of all inputs.
+	inputTotal btcutil.Amount
```

Comment:
can just loop over current inputs to get this instead of storing it as a var?

Reply:
This struct already tracks several running counters. Why would we recalculate `inputTotal` specifically every time we need it?

Reply:
Resolving this comment because it is the same as the one below